script.js

```diff
@@ -26,22 +26,32 @@ getLocationButton.addEventListener(`click`, event => {
   }
 });
 
+function clearModalData(event){
```

Comment:
I did this slightly differently 
`  const restaurantContent = restaurantCard.querySelector(
    ".inside-restaurant-info"
  ).cloneNode(true);
  const modalContent = document.querySelector("#modal-content");
    modalContent.append(restaurantContent);

  restaurantInfoModal.classList.add("modal-open");
}

function closeModal() {
  
  restaurantInfoModal.classList.remove("modal-open");
  const modalContent = document.querySelector("#modal-content");
  modalContent.innerHTML = "";
}`